xml/System.Numerics/Vector.xml

```diff
@@ -518,11 +525,13 @@
         <Parameter Name="value" Type="System.Numerics.Vector&lt;T&gt;" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of the input vector.</typeparam>
+        <param name="value">The vector to reinterpret.</param>
+        <summary>Reinterprets a <see cref="T:System.Numerics.Vector`1" /> as a new <see cref="T:System.Numerics.Vector`1" />.</summary>
```

Comment:
Should this be the same as the others, i.e.:

```suggestion
        <summary>Reinterprets the bits of a specified vector into those of a vector of native-sized integers.</summary>
```

Reply:
Yes, makes sense.